test/test.js

```diff
@@ -8,7 +8,7 @@
  * Module dependencies.
  */
 var should 		= require('should');
-var OO 			= require('../index');
+var OO 			= require('../dist/index').default;
```

Comment:
It's though if we can't keep coverage up to date.
I believe the only real problem with that is the export/import, the rest is completely supported with the latest Node.js v7 version, so how about we do move out export/import in-favor of the old Node require() and this will solve deprecating babel-polyfill inside the library code (and moving it out to webpack config), and also tests would run just the same.